packages/terra-toggler/src/Toggler.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Button from 'terra-button';
+import 'terra-base/lib/baseStyles';
+import './Toggler.scss';
+
+const propTypes = {
+  /**
+   * Content in the body of the panel that will be expanded or collapsed
+   **/
+  children: PropTypes.node,
+  /**
+   * Callback function when the toggler is opened.
+   **/
+  onOpen: PropTypes.func,
+  /**
+   * Callback function when the toggler is closed.
+   **/
+  onClose: PropTypes.func,
+  /**
+   * Content in the ‘header’ section that acts as a trigger for the collapse/expand action
+   **/
+  header: PropTypes.node.isRequired,
+  /**
+   * Animates expanding and collapsing
+   **/
+  isAnimated: PropTypes.bool,
+  /**
+   * Expands or collapses content
+   **/
+  isOpened: PropTypes.bool,
```

Comment:
so is the new standard to use `isOpened` vs `isOpen`
https://github.com/cerner/terra-core/blob/master/packages/terra-modal/src/Modal.jsx#L41
vs
https://github.com/cerner/terra-core/blob/master/packages/terra-slide-panel/src/SlidePanel.jsx#L41